packages/experience/src/pages/Consent/OrganizationSelector/OrganizationItem/index.tsx

```diff
@@ -21,27 +23,27 @@ const OrganizationItem = ({
   onSelect,
   isSelected,
   suffixElement,
+  className,
 }: OrganizationItemProps) => {
   return (
     <div
-      className={styles.organizationItem}
+      className={classNames(styles.organizationItem, className)}
       data-selected={isSelected}
       {...(onSelect && {
         role: 'button',
         tabIndex: 0,
         onClick: () => {
           onSelect(organization);
         },
-        onKeyDown: () => {
-          onKeyDownHandler({
-            Enter: () => {
-              onSelect(organization);
-            },
-            ' ': () => {
-              onSelect(organization);
-            },
-          });
-        },
+        onKeyDown: onKeyDownHandler({
+          Enter: () => {
+            console.log('enter');
+            onSelect(organization);
+          },
+          ' ': () => {
+            onSelect(organization);
```

Comment:
Also I think the `onKeyDownHandler` deals with space and enter keys by default, if you only pass the callback function.

<img width="756" alt="image" src="https://github.com/logto-io/logto/assets/12833674/2d371741-e870-42a5-8584-62ef13198007">

Reply:
cool updated